### PR TITLE
Feature/create formatAmount and parseAmount function to replace the deprecated ones denominate and nominate

### PR DESCRIPTION
**Pull request by CiprianDraghici**

### CODE REVIEW

**Review by ioandanpopa — APPROVED** [review]

File: src/utils/operations/denominate.ts [hunk 1]
```diff
@@ -21,122 +22,16 @@ export function denominate({
...
-
-          .valueOf();
-
-        return formattedBalance;
-      })
-      .if(isNegative)
-      .then((current) => `-${current}`)
-
-      .valueOf()
+  console.warn(
```
Comment (arhtudormorar) [inline src/utils/operations/denominate.ts:25]: use a variable above in the file to show the warning only once per app not to bloat the console with warnings

File: src/utils/operations/denominate.ts [hunk 2]
```diff
@@ -1,11 +1,12 @@
-import { TokenPayment } from '@elrondnetwork/erdjs';
 import BigNumber from 'bignumber.js';
-import { DIGITS, DECIMALS } from 'constants/index';
-import { stringIsInteger } from 'utils/validation/stringIsInteger';
-import { pipe } from './pipe';
+import { DECIMALS, DIGITS } from 'constants/index';
+import { formatAmount } from './formatAmount';
 
 BigNumber.config({ ROUNDING_MODE: BigNumber.ROUND_FLOOR });
```
Comment (arhtudormorar) [inline src/utils/operations/denominate.ts:5]: maybe this is no longer necessary here

File: src/utils/operations/nominate.ts [hunk 3]
```diff
@@ -1,6 +1,13 @@
 import { TokenPayment } from '@elrondnetwork/erdjs';
 import { DECIMALS } from 'constants/index';
 
+/**
```
Comment (arhtudormorar) [inline src/utils/operations/nominate.ts:4]: add `hasPrintedWarning = false`

File: src/utils/operations/nominate.ts [hunk 4]
```diff
@@ -1,6 +1,13 @@
...
+ * !!! This function is deprecated. Please use parseAmount instead.
+ * */
 export function nominate(amount: string, numDecimals: number = DECIMALS) {
+  console.warn(
```
Comment (arhtudormorar) [inline src/utils/operations/nominate.ts:8]: if (! hasprintedwarning) console.warn; hasPrinted = true

File: src/utils/operations/parseAmount.ts [hunk 5]
```diff
@@ -0,0 +1,7 @@
+import { DECIMALS } from 'constants/index';
+import { nominate } from './nominate';
+
+export function parseAmount(amount: string, numDecimals: number = DECIMALS) {
+  console.warn();
```
Comment (arhtudormorar) [inline src/utils/operations/parseAmount.ts:5]: see here that you are using nominate as primary function and will print warnings. Move nominate logic here

Comment (CiprianDraghici) [reply src/utils/operations/parseAmount.ts:5]: you are right.

File: src/utils/operations/denominate.ts (hunk 2 above)
Comment (CiprianDraghici) [reply src/utils/operations/denominate.ts]: are you sure ?

**Review by arhtudormorar — APPROVED** [review]

**Review by StanislavSava — APPROVED** [review]

**Pull request merged**